angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.component.ts

```diff
@@ -221,6 +221,9 @@ export class SprkMastheadSelectorComponent implements OnChanges {
    */
   @Input()
   triggerText: string;
+  /** Applies styles if the selector is flush with the sides of the viewport. */
+  @Input()
+  isFlush: boolean;
```

Comment:
I'm not gonna split hairs here: Renaming this to something that could be more descriptive as to what it does could make this easier to understand what it's doing in the long run.

- `isNarrowLayout`
- `hasNarrowStyles`

or something like that.